src/kinsol/kinsol_impl.h

```diff
@@ -399,6 +400,10 @@ void KINFreeAA(KINMem kin_mem);
 int KINInitOrth(KINMem kin_mem);
 void KINFreeOrth(KINMem kin_mem);
 
+/* Utility function to tell KINSOL to free the user data.
+   This is used by the Python interfaces. */
+int KINSetOwnUserData(void* kinmem, sunbooleantype own_user_data);
```

Comment:
Like IDA, KINSOL is inconsistent about this but to follow the general convention prefix with `kin`
```suggestion
int kinSetOwnUserData(void* kinmem, sunbooleantype own_user_data);
```